.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,17 +1,48 @@
-**What does this PR do / Why do we need it**:
+**What this PR does / why we need it**:
 
-**Does this PR close any issues?**:<!-- optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged -->
+**Which issue(s) this PR fixes**:
+<!--optional, in `fixes #<issue number>` format, will close the issue(s) when PR gets merged-->
 Fixes #
 
-**Special notes for your reviewer**:
+**What type of PR is this?**
+<!--
+Add one of the following kinds:
+/kind bug
+/kind cleanup
+/kind documentation
+/kind feature
+/kind design
 
-**Documentation**:
-<!-- Add links to the related documentation changes related to this pull request. E.g. the link to the kubermatic/docs pull request. -->
+Optionally add one or more of the following kinds if applicable:
+/kind api-change
+/kind deprecation
+/kind failing-test
+/kind flake
+/kind regression
+/kind question
```

Comment:
I don't think `/kind question` is applicable since no PR should be a question. Can we remove it?

Reply:
I removed it